src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/regler/vilkår/PassBarnRegel.kt

```diff
@@ -9,12 +16,52 @@ import no.nav.tilleggsstonader.sak.vilkår.regler.SvarId
 import no.nav.tilleggsstonader.sak.vilkår.regler.Vilkårsregel
 import no.nav.tilleggsstonader.sak.vilkår.regler.jaNeiSvarRegel
 import no.nav.tilleggsstonader.sak.vilkår.regler.regelIder
+import java.time.LocalDate
+import java.util.UUID
 
 class PassBarnRegel : Vilkårsregel(
     vilkårType = VilkårType.PASS_BARN,
     regler = setOf(HAR_ALDER_LAVERE_ENN_GRENSEVERDI, UNNTAK_ALDER, DEKKES_UTGIFTER_ANNET_REGELVERK, ANNEN_FORELDER_MOTTAR_STØTTE, UTGIFTER_DOKUMENTERT),
     hovedregler = regelIder(HAR_ALDER_LAVERE_ENN_GRENSEVERDI, DEKKES_UTGIFTER_ANNET_REGELVERK, ANNEN_FORELDER_MOTTAR_STØTTE, UTGIFTER_DOKUMENTERT),
 ) {
+
+    override fun initiereDelvilkår(metadata: HovedregelMetadata, resultat: Vilkårsresultat, barnId: UUID?): List<Delvilkår> {
+        val personIdentForGjeldendeBarn = metadata.barn.firstOrNull { it.id == barnId }?.ident
```

Comment:
Hvis man har resultat == SKAL_IKKE_VURDERES burde man kanskje ikke gjøre noe av dette? 
```suggestion
        if(resultat == Vilkårsresultat.SKAL_IKKE_VURDERES) {
            return super.initiereDelvilkår(metadata, resultat, barnId)
        }
        val personIdentForGjeldendeBarn = metadata.barn.firstOrNull { it.id == barnId }?.ident
```